deps/v8/include/v8-debug.h

```diff
@@ -20,7 +20,8 @@ enum DebugEvent {
   BeforeCompile = 4,
   AfterCompile = 5,
   CompileError = 6,
-  AsyncTaskEvent = 7,
+  PromiseEvent = 7,
```

Comment:
Looking at this more closely, I think the removal of `PromiseEvent` was actually an API change that is missing from the [official list](https://docs.google.com/document/d/1g8JFi8T_oAE_7uAri7Njtig7fKaPDfotU6huOa1alds/edit?hl=en&forcehl=1) (/cc @natorion). I think we will have bring this back as a revert of https://codereview.chromium.org/1833563002.